todo-mvc/package.json

```diff
@@ -23,6 +27,7 @@
     "grunt-text-replace": "^0.4.0",
     "grunt-ts": "^5.5.1",
     "grunt-tslint": "^3.1.0",
+    "grunt-typings": "^0.1.5",
```

Comment:
Why this versus [@types/grunt](https://www.npmjs.com/package/@types/grunt) and ideally shouldn't this be part of `grunt-dojo2` and then be unnecessary to be added here?